src/explorer/SubscriptionTreeItem.ts

```diff
@@ -109,6 +109,15 @@ export class SubscriptionTreeItem extends SubscriptionTreeItemBase {
         // site is set as a result of SiteCreateStep.execute()
         const siteClient: SiteClient = new SiteClient(nonNullProp(wizardContext, 'site'), this.root);
 
+        // https://github.com/microsoft/vscode-azureappservice/issues/840
+        if (wizardContext.newSiteOS === WebsiteOS.windows) {
+            const appSettings: StringDictionary = await siteClient.listApplicationSettings();
+            if (appSettings.properties && appSettings.properties.WEBSITE_NODE_DEFAULT_VERSION) {
+                delete appSettings.properties.WEBSITE_NODE_DEFAULT_VERSION;
```

Comment:
Are we sure it should be removed? This article seems to imply it has an effect... maybe we just pick a better default than 6.9.1?

https://blogs.msdn.microsoft.com/azureossds/2018/09/25/how-to-set-node-version-for-windows-node-js-webapp-from-azure-portal/